src/layman/layer/rest_layer_style.py

```diff
@@ -0,0 +1,46 @@
+from flask import Blueprint, Response, current_app as app, g
```

Comment:
File `src/layman/layer/rest_workspace_layer_style.py` should be removed in this commit, but it is not.